test/common_utils.py

```diff
@@ -638,6 +638,7 @@ def make_bounding_box(
     dtype=None,
     device="cpu",
 ):
+    batch_dims = ()  # This is nasty but this whole thing will be removed soon.
```

Comment:
@pmeier I had to add this to fix the tests. I'm sure there's a better way to fix it, but I don't want to spend time on this (already spent 10+ minutes) as these tests an helpers will be removed.

Feel free to push a better fix if you prefer.

Reply:
> as these tests an helpers will be removed.

You put the hack into `make_bounding_box`, which will not be removed. I've moved it into the helpers that will actually be removed. Nothing new is failing after that. Meaning, we are not misusing `make_bounding_box` anywhere from the new side of the tests.